cmd/kubeadm/app/preflight/checks.go

```diff
@@ -1091,6 +1117,15 @@ func RunRootCheckOnly(ignorePreflightErrors sets.Set[string]) error {
 	return RunChecks(checks, os.Stderr, ignorePreflightErrors)
 }
 
+// RunKernelVersionCheck initializes checks slice of structs and call RunChecks
+func RunKernelVersionCheck(ignorePreflightErrors sets.Set[string]) error {
```

Comment:
How about renaming it `RunUpgradeChecks` which contains the `IsPrivilegedUserCheck` and `KernelVersionCheck`?

Reply:
Sounds good, Let met try.